stackslib/src/monitoring/mod.rs

```diff
@@ -46,19 +47,20 @@ pub fn increment_rpc_calls_counter() {
 }
 
 pub fn instrument_http_request_handler<F, R>(
+    conv_http: &mut ConversationHttp,
     req: StacksHttpRequest,
```

Comment:
You need `mut` here when building with `monitoring_prom`, but it will warn when you aren't (like when running `cargo test`)

Reply:
It would be best just to make two separate versions of `instrument_http_request_handler()`, so we could take the multiple `#[cfg(feature = "monitoring_prom")]` lines out of the function body

Reply:
ah, oops. let me just delete that commit.